Core/scripts/dirac-configure.py

```diff
@@ -400,9 +400,13 @@ def setExtensions(optionValue):
     result = bdc.syncCAs()
     if result['OK']:
       result = bdc.syncCRLs()
-  except BaseException:
+    if not result['OK']:
+      DIRAC.gLogger.warn('Failed to sync CAs and CRLs: %s' % result['Message'])
```

Comment:
The `warn` function has a tendency to not be printed by default when used in scripts, which use `notice` as the visible log level threshold. One could use either `notice` or `error`